pkg/webui/console/views/application-data/index.js

```diff
@@ -0,0 +1,66 @@
+  { name: 'gs.down.send', time: '2019-04-15T09:20:39.672510Z', identifiers: [{ gateway_ids: { gateway_id: 'admin-gtw', eui: '0102030405060708' }}], data: { '@type': 'type.googleapis.com/ttn.lorawan.v3.DownlinkMessage', raw_payload: 'IJJfgSo3PVroy3IL9z9MUwojoT9Yi0CtUaRv1xdIXIiE', scheduled: { data_rate: { lora: { bandwidth: 125000, spreading_factor: 12 }}, coding_rate: '4/5', frequency: '868100000', timestamp: 717249018, downlink: { tx_power: 16.15, invert_polarization: true }}, correlation_ids: [ 'gs:conn:01D8G4RSYNH95CEQ0CY3Y6ESH8', 'gs:uplink:01D8G4RT0SJK80Y33195AP8YK4', 'ns:downlink:01D8G4RT7QHVG8G489ES6K2QJD', 'ns:uplink:01D8G4RT0VVYWXB188X8FXJEV8', 'rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01D8G4RT0T74QP3GXGH3G0KGDA', 'rpc:/ttn.lorawan.v3.GtwGs/LinkGateway:01D8G4RSVNGBNQMGZ3Q1QJQFKK', 'gs:conn:01D8G4RSYNH95CEQ0CY3Y6ESH8', 'rpc:/ttn.lorawan.v3.GtwGs/LinkGateway:01D8G4RSVNGBNQMGZ3Q1QJQFKK', 'rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01D8G4RT7RR7D5FSZGV7YTVMEY' ]}, correlation_ids: [ 'gs:conn:01D8G4RSYNH95CEQ0CY3Y6ESH8', 'rpc:/ttn.lorawan.v3.GtwGs/LinkGateway:01D8G4RSVNGBNQMGZ3Q1QJQFKK', 'rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01D8G4RT7RR7D5FSZGV7YTVMEY', 'gs:conn:01D8G4RSYNH95CEQ0CY3Y6ESH8', 'gs:uplink:01D8G4RT0SJK80Y33195AP8YK4', 'ns:downlink:01D8G4RT7QHVG8G489ES6K2QJD', 'ns:uplink:01D8G4RT0VVYWXB188X8FXJEV8', 'rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01D8G4RT0T74QP3GXGH3G0KGDA', 'rpc:/ttn.lorawan.v3.GtwGs/LinkGateway:01D8G4RSVNGBNQMGZ3Q1QJQFKK', 'gs:conn:01D8G4RSYNH95CEQ0CY3Y6ESH8', 'rpc:/ttn.lorawan.v3.GtwGs/LinkGateway:01D8G4RSVNGBNQMGZ3Q1QJQFKK', 'rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01D8G4RT7RR7D5FSZGV7YTVMEY' ], origin: 'htdvissermbp' },
+  { name: 'gs.gateway.disconnect', time: '2019-04-15T09:20:39.679935Z', identifiers: [{ gateway_ids: { gateway_id: 'admin-gtw', eui: '0102030405060708' }}], correlation_ids: [ 'gs:conn:01D8G4RSYNH95CEQ0CY3Y6ESH8', 'rpc:/ttn.lorawan.v3.GtwGs/LinkGateway:01D8G4RSVNGBNQMGZ3Q1QJQFKK' ], origin: 'htdvissermbp' },
+]
+
+export default class Data extends React.Component {
```

Comment:
I believe this whole file is not supposed to be here, or at least not in this stage?